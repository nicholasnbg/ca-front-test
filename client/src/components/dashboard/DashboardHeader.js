import React from "react";
import styled from "styled-components";

export default function DashboardHeader() {
  return (
    <Row>
      <ColText>Survey Details</ColText>
      <ColText>Response Rate</ColText>
    </Row>
  );
}

const Row = styled.div`
  background: #ffffff;
  border-radius: 10px 10px 0 0;
  border-bottom: 1px solid #d5d5d5;
  padding: 10px 0;
  width: 100%;
  display: grid;
  grid-template-columns: 3fr 1fr 0.5fr;
  @media (max-width: 700px) {
    grid-template-columns: 1fr 1fr 0.5fr;
  }
`;

const ColText = styled.em`
  padding-left: 25px;
`;
