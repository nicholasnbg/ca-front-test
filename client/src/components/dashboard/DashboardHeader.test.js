import React from "react";
import renderer from "react-test-renderer";

import DashboardHeader from "./DashboardHeader";

it("renders correctly", () => {
  const tree = renderer.create(<DashboardHeader />).toJSON();
  expect(tree).toMatchSnapshot();
});
