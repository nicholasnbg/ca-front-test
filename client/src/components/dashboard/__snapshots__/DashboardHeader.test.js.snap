// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="sc-bdVaJa ePNJYe"
>
  <em
    className="sc-bwzfXH iguHiQ"
  >
    Survey Details
  </em>
  <em
    className="sc-bwzfXH iguHiQ"
  >
    Response Rate
  </em>
</div>
`;
